src/imageLoader/colorSpaceConverters/convertPALETTECOLOR.js

```diff
@@ -54,4 +61,6 @@ export default function (imageFrame, rgbaBuffer) {
     rgbaBuffer[rgbaIndex++] = bDataCleaned[value];
     rgbaBuffer[rgbaIndex++] = 255;
   }
+
+  return rgbaBuffer;
```

Comment:
Why you are adding a return, since I don't see it being used on the return, it is just `await convertColoSpace`